policyengine_us/variables/gov/states/az/tax/income/deductions/itemized/az_itemized_deduction.py

```diff
@@ -0,0 +1,72 @@
+from policyengine_us.model_api import *
+
+
+class az_itemized_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Itemized Deductions"
+    unit = USD
+    documentation = "Arizona Form 140 Schedule A"
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+    reference = (
+        "https://law.justia.com/codes/arizona/2022/title-43/section-43-1042/",
+        "https://azdor.gov/forms/individual/itemized-deduction-adjustments-form",
+        "https://azdor.gov/forms/individual/form-140-resident-personal-income-tax-form-calculating",
+    )
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.irs.deductions
+        deductions = [
+            deduction
+            for deduction in p.itemized_deductions
+            if deduction
+            not in [
+                "salt_deduction",
+                "medical_expense_deduction",
+                "charitable_deduction",
+            ]
+        ]
+        federal_deductions = add(tax_unit, period, deductions)
+        # Adjustment to Medical and Dental Expresses
+        medical_expense = add(tax_unit, period, ["medical_expense"])
+        medical_expense_deduction = tax_unit(
+            "medical_expense_deduction", period
+        )
+        az_medical_expense_deduction = where(
```

Comment:
This is equivalent to the code you wrote, but is it correct? You already skipped the medical expense deduction in the for loop, I think you can just add the entirety of medical_expense
```suggestion
        az_medical_expense_deduction = np.abs(medical_expense - medical_expense_deduction)
```

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/9f82afdf-6f68-4d70-b8de-656059abdf03)

Arizona seems to exclude the AGI floor

Reply:
Right, excluding the federal deduction and adding all expenses will reflect that